the-experts-BE/routes/contractor-results.js

```diff
@@ -0,0 +1,13 @@
+var express = require("express");
+var router = express.Router();
+var queries= require("../queries/index")
+router.get("/", function(req, res, next) {
+    console.log("hellow world")
+    queries.getContractor(req.params.job,(err,response)=>{
+        if(err) console.log(err);
```

Comment:
Why are you just `console.log()` this error? You have error handling middleware setup, use `next()` to pass the error to it